apps/sms/test/marionette/conversation_test.js

```diff
@@ -314,4 +314,17 @@ marionette('Conversation Panel Tests', function() {
       assert.equal(downloadedMessage.content.trim(), 'Attachment');
     });
   });
+
+  suite('Edit Mode', function() {
+    var thread, conversationView;
+    setup(function() {
+      thread = ThreadGenerator.generate({
+        numberOfMessages: 10
+      });
+      messagesApp.launch();
+      storage.setMessagesStorage([thread], ThreadGenerator.uniqueMessageId);
+    });
+    var inboxView = new InboxView(client);
```

Comment:
Have you read the links about ```setup```, ```test``` and ```teardown``` I've mentioned in the previous comment?  You need better understand what they are and where you place your code :)

Having said that the last two statements are declared literally nowhere in the test flow.

Once you deal with that you need to see into lib/views folders and understand ```accessor.js``` and ```view.js``` for existing views, to feel the difference. Then inspect header in edit mode (when you enter conversation and choose "..." -> Select messages header is changed) in browser to find out what selectors are assigned to edit header buttons (Select All, Deselect all and etc.). And finally expose these buttons via view and accessor